pkgs/racket-doc/scribblings/reference/match.scrbl

```diff
@@ -202,7 +203,7 @@ In more detail, patterns match as follows:
          the patterns @racket[(list-no-order x 1 x)] and
          @racket[(list-no-order x 1 x ...)] both produce syntax errors.}}
 
- @item{@racket[(#,(racketidfont "list-no-order") _pat ... _lvp)] ---
+ @item{@racket[(#,(match-kw "list-no-order") _pat ... _lvp)] ---
```

Comment:
This is the second indexing. Is it OK? (It could be OK, but I'm not sure)

Reply:
Ah, that was not intended.

Reply:
```suggestion
 @item{@racket[(#,(racketidfont "list-no-order") _pat ... _lvp)] ---
```